docs_template/source/getting_started/preparing_your_system.md

```diff
@@ -3,14 +3,10 @@
 Make sure you have the following programs installed and that these can be found on your
 path. This template requires
 
-- [Miniconda](https://docs.conda.io/en/latest/miniconda.html) or
-  [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html) or
-  Anaconda. Windows users: please consult {ref}`windows_user`
+- Python, which you must install using the installation instructions
+  [here](https://effective-programming-practices.vercel.app/python_installation_execution/index.html).
```

Comment:
Please link directly to the pages for Windows, MacOS, Linux, and WSL for the adventurous.